### PR TITLE
adding the rest of the base downtown deck

### PR DESCRIPTION
**Pull request by ThomasLipp**

### CODE REVIEW

File: eldritch/encounters.py [hunk 1]
```diff
@@ -151,17 +151,119 @@ def Asylum1(char):
   roll3 = events.Gain(char, {"clues": 3})
   cond = events.Conditional(char, check, "successes", {0: roll0, 1: roll1, 3: roll3})
   return events.Sequence([check, cond], char)
+def Asylum2(char):
+  check = events.Check(char, "speed", -1)
+  item = events.Draw(char, "unique", 1)
+  move = events.ForceMovement(char, "Downtown")
+  cond = events.Conditional(char, check, "successes", {0: move, 1: item})
```
Comment (fingoldfish) [inline eldritch/encounters.py:158]: You never use this conditional.

File: eldritch/encounters.py [hunk 2]
```diff
@@ -151,17 +151,119 @@ def Asylum1(char):
...
+  return events.Nothing() # TODO: implement location choice
+def Bank2(char):
+  check = events.Check(char, "luck", -1)
+  spend = events.Loss(char, {"dollars": 2})
+  common = events.Draw(char, "common", 1)
+  unique = events.Draw(char, "unique", 1)
+  cond = events.Conditional(char, check, "successes", {0: common, 1: unique})
+  prereq = events.AttributePrerequisite(char, "dollars", 2, "at least")
+  nothing = events.Nothing()
+  choice = events.BinaryChoice(char, "Pay $2 for man's last possession?",
```
Comment (fingoldfish) [inline eldritch/encounters.py:197]: When the arg list is very long, I prefer to have nothing on the line with the open paren, and just start a new line.

File: eldritch/encounters.py [hunk 3]
```diff
@@ -151,17 +151,119 @@ def Asylum1(char):
...
 
 def Bank1(char):
-    return events.Nothing() # TODO: implement location choice
+  return events.Nothing() # TODO: implement location choice
+def Bank2(char):
+  check = events.Check(char, "luck", -1)
+  spend = events.Loss(char, {"dollars": 2})
+  common = events.Draw(char, "common", 1)
+  unique = events.Draw(char, "unique", 1)
+  cond = events.Conditional(char, check, "successes", {0: common, 1: unique})
```
Comment (fingoldfish) [inline eldritch/encounters.py:194]: You could also replace this with a PassFail, but it's fine as-is.

File: eldritch/encounters.py [hunk 4]
```diff
@@ -151,17 +151,119 @@ def Asylum1(char):
...
 def Square1(char):
   return events.Gain(char, {"stamina": 1})
+def Square2(char):
+  check = events.Check(char, "will", -1)
+  # TODO: prerequisite ally being in the deck, otherwise two clue tokens
+  ally = events.DrawSpecific(char, "allies", "Fortune Teller")
+  return events.PassFail(char, check, ally, events.Nothing())
+def Square3(char):
+  check = events.Check(char, "will", -1)
+  loss = events.Loss(char, {'sanity': 1, 'stamina': 1})
```
Comment (fingoldfish) [inline eldritch/encounters.py:232]: To keep things consistent, use double quotes here instead of single quotes.

File: eldritch/places.py [hunk 5]
```diff
@@ -91,8 +91,8 @@ def CreatePlaces():
   Shop = Location("Shop", "Curiositie Shoppe", None)
   Northside = Street("Northside", "Northside")
   Northside._add_connections(Newspaper, Train, Shop)
-  Bank = Location("Bank", "Bank", None)
-  Asylum = Location("Asylum", "Asylum", None)
+  Bank = Location("Bank", "Bank of Arkham", None)
```
Comment (fingoldfish) [inline eldritch/places.py:94]: The word Arkham should not appear in the source code.

**Review by fingoldfish — APPROVED** [review]

File: eldritch/encounters.py (hunk 1 above)
Comment (ThomasLipp) [reply eldritch/encounters.py]: yeah , realized that I didn't need to use a conditional for everything. removed

File: eldritch/encounters.py (hunk 2 above)
Comment (ThomasLipp) [reply eldritch/encounters.py]: done

File: eldritch/encounters.py (hunk 3 above)
Comment (ThomasLipp) [reply eldritch/encounters.py:193]: leaving as is

File: eldritch/encounters.py (hunk 4 above)
Comment (ThomasLipp) [reply eldritch/encounters.py]: at spaceX almost everything tends to be single quotes, so I'm having to train myself

File: eldritch/places.py (hunk 5 above)
Comment (ThomasLipp) [reply eldritch/places.py]: removed

**Pull request merged**